src/generateNumber.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+function generateRandomNumber() {
+  const min = 1000;
+  const max = 9999;
+
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+const value = generateRandomNumber().toString();
```

Comment:
Export the function, not the number, what if we wanted to play again. The function is called generateNumber.js, not randomNumber.js after all :)